INSTALL.rst

```diff
@@ -87,65 +89,80 @@ For example, on Debian GNU/Linux, you can install them as follows:
                         rabbitmq-server \
                         redis-server
 
-and do a system install for the Sass preprocessor by following `Sass web guide <http://sass-lang.com/install>`_ and running:
+Now, add the following lines in your "elasticsearch.yml" (for
+Debian GNU/Linux the full path is
+``/etc/elasticsearch/elasticsearch.yml``):
 
 .. code-block:: shell
 
-  sudo npm install -g sass node-sass clean-css uglify-js requirejs
-
+  # MY CONFIGS
 
-Installation
-^^^^^^^^^^^^
+  cluster.name: cap
+  discovery.zen.ping.multicast.enabled: false
+  http.port: 9200
+  http.publish_port: 9200
 
-Let's start by creating a new virtual environment that will hold our CAP
-v3.0 instance:
+Finally, do a system install (see below for how to do a local install
+instead) for the Sass preprocessor by following
+`Sass web guide <http://sass-lang.com/install>`_ and running:
 
 .. code-block:: shell
 
-   mkvirtualenv cap
+  sudo npm install -g node-sass clean-css uglify-js requirejs
 
-Start redis server in the background:
+Installation
+^^^^^^^^^^^^
+
+Let's start by cloning the repository:
```

Comment:
Then if you want this way it is better to use ```git-new-workdir``` like so http://nuclearsquid.com/writings/git-new-workdir/ or better the scripts from git that supersedes it ```git worktree``` (https://git-scm.com/docs/git-worktree)

It used to be the old invenio way. I am not using this method, so I can't advise further